btb/examples/random_forest.py

```diff
@@ -0,0 +1,57 @@
+from sklearn.datasets import fetch_mldata
```

Comment:
According to PEP8, the imports should be organized in 3 blocks, separated by one blank line:
* stdlib imports
* third party packages (like sklearn or numpy)
* project modules (btb.*)

Also, within each block, all the `import module_name` imports should go before the `from module_name import object_name` ones.

A part from this, I personally recommend to also sort the imports alphabetically within each block. This helps a lot preventing duplicates when the number of imports grows.

This actually applies to all the modules in the commit.